apps/condo/domains/marketplace/schema/MarketItemPrice.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Generated by `createschema marketplace.MarketItemPrice 'price:Json; marketItem:Relationship:MarketItem:CASCADE; organization:Relationship:Organization:CASCADE;'`
+ */
+
+const get = require('lodash/get')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema  } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/marketplace/access/MarketItemPrice')
+const { PRICE_FIELD } = require('@condo/domains/marketplace/schema/fields/price')
+
+
+const ERRORS = {
+    INVALID_SALES_TAX_PERCENT: (rowNumber) => ({
+        code: BAD_USER_INPUT,
+        type: 'INVALID_SALES_TAX_PERCENT',
+        messageForUser: 'api.marketplace.MarketItemPrice.error.INVALID_SALES_TAX_PERCENT',
+        messageInterpolation: { rowNumber },
+        message: `Error INVALID_SALES_TAX_PERCENT on line ${rowNumber}. Must be grater when -1 and less or equal when 100.`,
+    }),
+    INVALID_PRICE: (rowNumber) => ({
+        code: BAD_USER_INPUT,
+        type: 'INVALID_PRICE',
+        messageForUser: 'api.marketplace.MarketItemPrice.error.INVALID_PRICE',
+        messageInterpolation: { rowNumber },
+        message: `Error INVALID_PRICE on line ${rowNumber}. Must be grater when -1.`,
+    }),
+}
+
+const MarketItemPrice = new GQLListSchema('MarketItemPrice', {
+    schemaDoc: 'MarketItemPrice is set separately from MarketItem and indicates the price of the service.',
+    fields: {
+
+        price: PRICE_FIELD,
+
+        marketItem: {
+            schemaDoc: 'Link to MarketItem',
+            type: 'Relationship',
+            ref: 'MarketItem',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+    },
+    hooks: {
+        validateInput: ({ resolvedData, existingItem, context }) => {
+            const nextData = { ...existingItem, ...resolvedData }
+            // Check rows
+            const prices = get(nextData, 'price', [])
+            for (let i = 0; i < prices.length; i++) {
+                if (Number(get(prices[i], 'salesTaxPercent', null)) < 0) {
```

Comment:
The tax percentage value can not be more than 100 :)
```suggestion
                const valueToCheck = Number(get(prices[i], 'salesTaxPercent', null))
                if (valueToCheck < 0 || valueToCheck > 100) {
```